examples/time_frequency/plot_compute_raw_data_spectrum.py

```diff
@@ -70,3 +71,18 @@
 
 ax.set_title('Four left-temporal magnetometers')
 plt.legend(['Without SSP', 'With SSP', 'SSP + Notch'])
+
+# Alternatively, you may also create PSDs from Raw objects with psd_XXX
+f, ax = plt.subplots()
+psds, freqs = psd_multitaper(raw, low_bias=True, tmin=tmin, tmax=tmax,
```

Comment:
> make_projector goes through that list and removes the bad channels

Actually setting bad channels is one way to implicitly subselect projection
operators. We should think if there's a good way to deal with it. But let's
drop this discussion here and start a new issue. We're drifting too far
afield...